code/modules/reagents/machinery/grinder.dm

```diff
@@ -1,130 +1,48 @@
 #define REAGENTS_PER_SHEET 20
 
 /obj/machinery/reagentgrinder
-	name = "All-In-One Grinder"
-	icon = 'icons/obj/kitchen.dmi'
-	icon_state = "juicer1"
-	layer = BELOW_OBJ_LAYER
-	density = 0
-	anchored = 0
 	use_power = 1
 	idle_power_usage = 5
 	active_power_usage = 100
-	circuit = /obj/item/weapon/circuitboard/reagentgrinder
-	var/inuse = 0
-	var/obj/item/weapon/reagent_containers/beaker = null
+	var/nano_template
 	var/limit = 10
 	var/list/holdingitems = list()
 	var/list/sheet_reagents = list(
 		/obj/item/stack/material/iron = "iron",
-		/obj/item/stack/material/uranium = MATERIAL_URANIUM,
+		/obj/item/stack/material/uranium = "uranium",
```

Comment:
What happened to this ?

Reply:
These defines are for materials, not reagents. It just happens that some reagents have the same name as a material.
This line refers to https://github.com/discordia-space/CEV-Eris/blob/ed2544a340855e1de7de51a1516ed0c4a55e419b/code/modules/reagents/reagents/other.dm#L169-L171 and not to https://github.com/discordia-space/CEV-Eris/blob/ed2544a340855e1de7de51a1516ed0c4a55e419b/code/modules/materials/materials.dm#L244-L245
Whoever added material defines must have placed them here by mistake.